bashhub/shell/deps/fish/functions/__bh_check_bashhub_installation.fish

```diff
@@ -0,0 +1,30 @@
+#
+# __bh_check_bashhub_installation.fish
+# Should only be sourced once following installation.
+#
+
+# Small function to check our Bashhub installation.
+function __bh_check_bashhub_installation
+    set -l ret 0
+    set -l config_path "$BH_HOME_DIRECTORY/config"
+    if [ -n "$FISH_VERSION" ] && [ -n "$__bh_enable_subshells" ] && [ "(trap)" -ne *"__bh_preexec_invoke_exec"* ]
+        echo "Bashhub's preexec hook is being overriden and is not saving commands. Please resolve what may be holding the DEBUG trap."
+        set ret 1
```

Comment:
We can remove this section as its intended to check bash's trap being used for preexec

Reply:
Ah, that makes sense. I've removed the check from that function.